storage/storage.go

```diff
@@ -1,9 +1,19 @@
 package storage
 
-import "io"
+import (
+	"io"
+	"os"
+)
+
+type File struct {
```

Comment:
Ah okay I misunderstood the struct. I don't like the name though. Maybe `FileEntry`?.

Also let me ask @bradrydzewski if there's something already around which is similar to what we want already in the library. Or maybe he would prefer something like a Walk function.